feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/service/network/impl/MetisModificationServiceImpl.kt

```diff
@@ -218,4 +225,33 @@ internal class MetisModificationServiceImpl(
                 .isSuccess()
         }
     }
+
+    override suspend fun uploadFileOrImage(
+        context: MetisContext,
+        courseId: Long,
+        conversationId: Long,
```

Comment:
The `context` parameter is currently not used. I suggest to remove the other parameters `courseId` and `conversationId` and change `context` to be of type `MetisContext.Conversation`. Then we can get the `courseId` and `conversationId` directly from the context.